### PR TITLE
Remove support for rockets

### PR DESCRIPTION
**Pull request by Guy-Adler**
This Pull Request implements the idea in #18.
It also disables the autopilot for gliders and hot air balloons.

### CODE REVIEW

**Comment by nicolas377** [comment]
Should we also include the fix for the keyboard mapping in 0.12.0?

**Comment by nicolas377** [comment]
oop, just noticed this was targeting main instead of release.
all good then!

File: source/ui/autopilot.ts [hunk 1]
```diff
@@ -6,7 +6,8 @@ define([
   "autopilot",
   "util",
   "getwaypoint",
```
Comment (nicolas377) [inline source/ui/autopilot.ts:8]: Shouldn't we include the array in the dependencies of this?

Comment (Guy-Adler) [reply source/ui/autopilot.ts:8]: Can you clarify what you mean by that?

Comment (nicolas377) [reply source/ui/autopilot.ts:8]: I just realized it's part of the addition, not before the diff subtraction. all good!

**Review by nicolas377 — APPROVED** [review]

**Pull request merged**